### PR TITLE
Feature/nsa 7151 update help pages to include new request a sub service information

### PR DESCRIPTION
**Pull request by stottidfe**
fixing the help ages to reflect the new features in services

### CODE REVIEW

File: src/app/manageUsers/changeSubServiceToUser.js
Comment (malikimran34) [inline src/app/manageUsers/changeSubServiceToUser.js:1]: can you please add a small unit test to touch the assert the return view. thanks

File: src/app/manageUsers/views/changeSubServiceToUser.ejs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            <li>Select the service the sub-service belongs to and click 'continue'.</li>            
+            <li>Select or deslect the sub-service that you want to add or remove and click 'continue'.</li>            
+            <li>Review the sub-service change for the end user and click 'approve'.</li>            
+            <li>A notification banner will confirm you have changed sub-service access for the end user.</li>
+        </ol>
+        <p class="govuk-body end-of-section">
+            An end user can also request changes to their sub-service access. Which an approver must either approve or reject.
+        </p>
+        <p class="govuk-body end-of-section">
+            Learn more about how to <a class="govuk-link" ref="#">approve a sub-service access request</a> and <a class="govuk-link" href="#">how to reject a sub-service request</a>.
```
Comment (malikimran34) [inline src/app/manageUsers/views/changeSubServiceToUser.ejs:36]: not sure how it should look? 
does it need a question mark in 'how to reject a sub-service request?'

File: src/app/requests/views/howToApprove.ejs [hunk 2]
```diff
@@ -8,18 +8,19 @@
...
             </h2>
             <p class="govuk-body">
-                There are two types of request you may be asked to approve: organisation requests, and service requests.
+                As an approver, there are 3 types of request you may be asked to approve:
+                <ul class="govuk-list govuk-list--bullet"><li>organisation requests</li><li>service requests</li><li>sub-service requests</li></ul>
             </p>
         </div>
 
         <!-- help page content -->
         <h3 id="how-to-approve-org-request" class="govuk-heading-m">How to approve an organisation request</h3>
```
Comment (malikimran34) [inline src/app/requests/views/howToApprove.ejs:17]: question mark ?

File: src/app/requests/views/howToApprove.ejs [hunk 3]
```diff
@@ -8,18 +8,19 @@
...
         <ol class="govuk-list govuk-list--number">
-            <li>Go to 'requests' (you can find this at the top in the navigation bar).</li>
+            <li>Go to 'requests'.</li>
             <li>Select the request you would like to approve or review.</li>
-            <li>Read the request review page carefully.</li>
+            <li>Review the request details.</li>
             <li>If you are confident the request should be approved, select 'approve'.</li>
-            <li>The user will receive an email informing them of their approval.</li>
+            <li>The user will receive an email confirmming of their approval.</li>
```
Comment (malikimran34) [inline src/app/requests/views/howToApprove.ejs:23]: typo in confirmming => 'confirming'

**Review by malikimran34 — APPROVED** [review]
Look for those typos and address them thank you

File: src/app/manageUsers/views/changeSubServiceToUser.ejs (hunk 1 above)
Comment (stottidfe) [reply src/app/manageUsers/views/changeSubServiceToUser.ejs:36]: not according to designs its a link text I did add a location to go to when they press the button though

File: src/app/requests/views/howToApprove.ejs (hunk 3 above)
Comment (stottidfe) [reply src/app/requests/views/howToApprove.ejs:23]: ni e one thanks

**Pull request merged**